### PR TITLE
Fix flake8 errors

### PR DESCRIPTION
**Pull request by csomh**
Mostly adding documentation and removing unused imports.
Resolving #7

### CODE REVIEW

File: releng_sop/kojibase.py [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 from .common import Release
 
+
 class KojiBase(object):
+    """Base class for koji commands."""
+
     def __init__(self, env, release_id, packages):
+        """Init.
+
+        Patameters
```
Comment (dmach) [inline releng_sop/kojibase.py]: Pa**r**ameters :)

File: releng_sop/koji_block_package_in_release.py [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
 
+Constructs the koji command
+
+    koji --profile=KOJI_PROFILE block-pkg RELEASE_TAG PACKAGES...
+
+KOJI_PROFILE is obtained from the environment settings.
+
+RELEASE_TAG is the "tag_release" key from the release configuration
+    corresponding to the release id.
+"""
```
Comment (dmach) [inline releng_sop/koji_block_package_in_release.py]: This partially overlaps with the "manual steps" section in docs.
Not sure if it's duplicity or not.

File: releng_sop/koji_block_package_in_release.py [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
+
     def print_details(self, commit=False):
+        """Print details of command execution.
+
+        Parameters
+        ----------
+        commit: boolean (optional; default False)
+            Flag to indicate if the command will be actually executed.
+            Line indicating "test mode" is printed, if this is False.
+        """
```
Comment (dmach) [inline releng_sop/koji_block_package_in_release.py]: I'd use sphinx syntax for this (:param:, :type:, ...)
See: https://github.com/release-engineering/productmd/blob/master/productmd/common.py#L81

File: releng_sop/koji_block_package_in_release.py (hunk 2 above)
Comment (csomh) [inline releng_sop/koji_block_package_in_release.py]: Would leave like this for now, #11 will solve it.

**Pull request merged**